src/Counter/README.md

```diff
@@ -16,17 +16,18 @@ A Counter is a number value that can be updated by two Buttons surrounding the d
 
```

Comment:
Do not hesitate to expand the description. `The value won't go beyond the min and max boundaries and will always increase or decrease with the same step.`